core/server/middleware/error-handler.js

```diff
@@ -4,9 +4,11 @@ var _ = require('lodash'),
     config = require('../config'),
     errors = require('../errors'),
     i18n = require('../i18n'),
-    _private = {};
+    _private = {},
+    errorHandler = {},
+    htmlStackExclusions = ['NotFoundError', 'DatabaseNotPopulatedError'];
 
-_private.parseStack = function (stack) {
+_private.parseStack = function parseStack(stack) {
```

Comment:
i have two questions:
1. any information to share what this function is doing?
2. can we put this function into the GhostError?

```
if (!err instanceof errors.NotFoundError && !err instanceof DatabaseNotPopulatedError)
  templateData.stack = err.getPrettyStack();
```